source/unified-test-format/unified-test-format.rst

```diff
@@ -3280,8 +3283,10 @@ Change Log
 :2021-08-30: Add ``hasServerConnectionId`` field to ``commandStartedEvent``,
              ``commandSuccededEvent`` and ``commandFailedEvent``.
 
-:2021-08-24: Test runners may create an internal MongoClient for each mongos.
+:2021-08-30: Test runners may create an internal MongoClient for each mongos.
```

Comment:
Feel free to leave this as "08-30" as merging so we don't have to deal with it appearing _after_ @benjirewis's entry that bumped to 1.6.0.